src/devices/Bmi088/AccelerometerRegister.cs

```diff
@@ -1,80 +1,80 @@
-﻿namespace Iot.Device.GroveBmi088Device
+﻿namespace Iot.Device.Bmi088Device
 {
     /// <summary>
     /// Registers for the accelerometer
     /// </summary>
-    public enum AccelerometerRegister : byte
+    internal enum AccelerometerRegister : byte
     {
         /// <summary>See datasheet</summary>
-        CHIP_ID = 0x00, // Default value 0x1E
+        ChipId = 0x00, // Default value 0x1E
```

Comment:
for internal enums, especially those matching the spec we do not require renaming, it's fine both renamed or original spec names. We only require that for public APIs